dm-setup/qwiklabs.jinja

```diff
@@ -60,32 +72,32 @@ resources:
     sockShop: "installed"
   metadata:
     dependsOn:
-    - {{ env['deployment'] }}-network
+    - {{ env['deployment'] }}-us-central1
 - name: east
   type: cluster.jinja
   properties:
     location: us-east4-b
     network: {{ env['deployment'] }}-network
-    subnetwork: {{ env['deployment'] }}-network
+    subnetwork: {{ env['deployment'] }}-us-east4
     purpose: workloads
     apiVersion: v1beta1
     kubernetesVersion: "1.10.4"
     monitoringService: monitoring.googleapis.com/kubernetes
     loggingService: logging.googleapis.com/kubernetes
   metadata:
     dependsOn:
-    - {{ env['deployment'] }}-network
+    - {{ env['deployment'] }}-us-east4
 - name: spinnaker
   type: cluster.jinja
   properties:
     network: {{ env['deployment'] }}-network
-    subnetwork: {{ env['deployment'] }}-network
-    location: us-central1-f
+    subnetwork: {{ env['deployment'] }}-us-central1
```

Comment:
Should we use  `$(ref.{{ env['deployment'] }}-us-central1.name)` here

Reply:
that's equal to `{{ env['deployment'] }}-us-central1` AFAIK

Reply:
Sorry should have clarified. Adding that would ensure that the subnet is created before the cluster.